### PR TITLE
Solution

### PR DESCRIPTION
**Pull request by Daniil-Pankieiev**

### CODE REVIEW

**Comment by Daniil-Pankieiev** [comment]
image

**Review by Katherine-Greg — APPROVED** [review]
Good Job! 👍

File: taxi/migrations/0001_initial.py [hunk 1]
```diff
@@ -9,7 +9,6 @@
 
 
 class Migration(migrations.Migration):
-
```
[the remark this answers is not in the document]
Comment (Daniil-Pankieiev) [reply taxi/migrations/0001_initial.py:12]: I used black to fix flake8 and black changes migration. I deleted them

**Review by LiudmylaKulinchenko — APPROVED** [review]
Good job 🔥

**Pull request closed without merging**